statistics/handle/handle.go

```diff
@@ -654,3 +673,20 @@ func (h *Handle) columnCountFromStorage(tableID, colID int64) (int64, error) {
 	}
 	return rows[0].GetMyDecimal(0).ToInt()
 }
+
+func (h *Handle) statsMetaByTableIDFromStorage(tableID int64, historyStatsExec sqlexec.RestrictedSQLExecutor) (version uint64, modifyCount, count int64, err error) {
```

Comment:
Why call it `save`? Shouldn't it be `load`?

Reply:
What do you mean by `save`?